docs/enhancements/backup-restore-support.md

```diff
@@ -0,0 +1,151 @@
+---
+title: backup-restore-support
+authors:
+  - "@danielerez"
+creation-date: 2024-08-08
+last-updated: 2024-08-08
+---
+
+# Backup/Restore Support
+
+## Summary
+
+The Assisted Installer should support backup/restore and disaster recovery scenarios, either using OADP (OpenShift API for Data Protection) for ACM (Advanced Cluster Management), or, using ZTP (Zero Touch Provisioning) flows.
+I.e. the assisted-service should be resilient in such scenarios which, for this context and effort, means that restored/moved spoke clusters should keep the same state and behave the same on the new hub cluster. 
+
+## Motivation
+
+Provide resiliency in the assisted-service for safe backup/restore flows, allowing spoke clusters to be used without any restriction after DR scenarios or moving between hubs.
+
+### Goals
+
+Spoke clusters with BMHs should function properly and have a consistent state after these scenarios: 
+* Spoke clusters restored on a passive hub with ACM/OADP functionality.
+* Spoke clusters moved between hubs using ZTP flows.
+
+### Non-Goals
+
+* Improve import day2 cluster solution by getting full host data (e.g. using an agent on existing nodes).
+
+## Proposal
+
+### User Stories
+
+#### Story 1
+
+As an ACM/OADP user, I want to backup an installed spoke cluster from an active hub, including the associated BMHs. Then, I want to safely restore the cluster on a passive hub, including the BMHs that should keep the same state. I.e. the restored cluster should behave exactly as in the active hub with no restrictions (support day2 operations, etc).
+
+#### Story 2
+
+As an ACM/OADP user, I want to backup an installed spoke cluster from an active hub.
+Then, I want to safely restore the cluster on a passive hub, without including the BMHs, and enable a GitOps tool (e.g ArgoCD) on the passive hub. If the GitOps tool is configured to apply the BMHs CRs, it should not affect the spoke cluster.
```

Comment:
I think with or without the BMHs in the backup is an implementation detail. I don't think the use cares what we back up or not as long as everything is usable in the end.